internshala.json

```diff
@@ -0,0 +1,32 @@
+{
+    "slug": "internshala",
+    "relevant-countries": [
+        "india"
+    ],
+    "name": "Internshala",
+    "runs": [
+        "Internshala",
+        "Internships",
+        "Training",
+        
+    ],
+    "address": "Scholiverse Educare Pvt. Ltd. B-610, Unitech Business Zone, Nirvana Country, South City 2, Gurgaon, India - 122018",

```

Comment:
Please delimit the parts of the address with newlines instead of commas. Also, the last line should just be the country.